client/src/plugins/libparsec/definitions.ts

```diff
@@ -691,6 +699,12 @@ export interface LibParsecPlugin {
         recipient: UserID,
         role: RealmRole | null
     ): Promise<Result<null, ClientWorkspaceShareError>>
+    clientInfo(
+        client: number
```

Comment:
Use `handle` instead of `client` for the variable name.

Reply:
Many functions of the API use `client: Handle` in `parameter`, need to be discussed before